core/handler/gear_action/gear_action.ex

```diff
@@ -39,11 +39,16 @@ defmodule AntikytheraCore.Handler.GearAction do
     %Conn{context: %Context{start_time: t_start, context_id: context_id}} = conn
     log_message_base = "#{CoreConn.request_info(conn)} from=#{sender_info}"
     encoding = Conn.get_req_header(conn, "accept-encoding") || "(none)" # To see whether client accepts gzip/deflate compression or not
-    Writer.info(logger, t_start, context_id, "#{log_message_base} START encoding=#{encoding}")
+    amzn_info =
+      case Conn.get_req_header(conn, "x-amzn-trace-id") do
+        nil -> ""
+        id  -> " amzn_trace_id=" <> id
+      end
+    Writer.info(logger, t_start, context_id, "#{log_message_base} START encoding=#{encoding}" <> amzn_info)
```

Comment:
Since it is not necessary to maintain backward compatibility of the log format, I thought that `amzn_trace_id=(none)` could be used as well as the existing log.
Is it useful when using `grep`?

Reply:
Fixed. 128ecd7ce8f55d78966822bfa6dae9a49f9cbcff